code/C#/DBDefsLib/Structs.cs

```diff
@@ -2,42 +2,39 @@
 
 namespace DBDefsLib
 {
-    public class Structs
```

Comment:
This is a library, so external users will be broken by these changes. Even if the name is technically wrong, I'd avoid the rename and maybe add an alias only.

Reply:
I've reverted that